wire-ios-request-strategy/Sources/Request Strategies/MLS/MLSRequestStrategy.swift

```diff
@@ -49,6 +49,8 @@ public final class MLSRequestStrategy: AbstractRequestStrategy {
             withManagedObjectContext: managedObjectContext,
             applicationStatus: applicationStatus
         )
+
+        configuration = [.allowsRequestsDuringSlowSync, .allowsRequestsWhileOnline]
```

Comment:
**question** : what was the default configuration before?

Reply:
[.allowsRequestsWhileOnline] is the default, which wasn't allowing us complete the slow sync since we try to join/create MLS self conversation during the slow sync.

This was "working" before since previously the slow sync wasn't waiting for this to complete previously.